ligo/transfers.mligo

```diff
@@ -1,20 +1,15 @@
 // SPDX-FileCopyrightText: 2021 Arthur Breitman
 // SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
 
-#define X_IS_FA2
-//#define Y_IS_FA2
-
 type token_id = nat
 
 #if X_IS_FA2
-[@inline] let const_x_token_id = 0n (* CHANGEME *)
 type x_contract_transfer = (address * (address * (token_id * nat)) list) list
 #else
 type x_contract_transfer = address * (address * nat)
 #endif
 
 #if Y_IS_FA2
-[@inline] let const_y_token_id = 0n (* CHANGEME *)
```

Comment:
Good points.

Regarding (1), I hadn't noticed that was the case, fixed now (see the fixup commit).

As for (2) I pushed an additional commit (the last one) to move the constant to the storage, all the ones that are not pragmas (and that aren't the result of a calculation).

Point (3) was actually not bad before this, because while it was true that in some cases those constants weren't used it's also the case that they'd just be ignored by LIGO at no additional cost.

It's a bit worse now that they are fixed in the storage type, but as we discussed it's probably not worth the trouble to have different storage types.